### ISSUE TITLE
Crashes on start if it can't connect to humblebundle

### ISSUE THREAD

**Issue by darrik** [td512/Humble-Trove-Downloader#4]
I forgot to white list the application in windows firewall and received this on start.
```
Problem signature:
  Problem Event Name:	CLR20r3
  Problem Signature 01:	Humble Trove Downloader.exe
  Problem Signature 02:	1.0.0.0
  Problem Signature 03:	919ef244
  Problem Signature 04:	System
  Problem Signature 05:	4.7.3062.0
  Problem Signature 06:	5ab952a7
  Problem Signature 07:	209a
  Problem Signature 08:	158
  Problem Signature 09:	N3CTRYE2KN3C34SGL4ZQYRBFTE4M13NB
  OS Version:	6.1.7601.2.1.0.256.48
  Locale ID:	1053
  Additional Information 1:	795f
  Additional Information 2:	795f1e36c63d49d93ac84e48469cee12
  Additional Information 3:	23f2
  Additional Information 4:	23f2cddac3732cdc34a81b0b878c3007

Read our privacy statement online:
  http://go.microsoft.com/fwlink/?linkid=104288&clcid=0x0409

If the online privacy statement is not available, please read our privacy statement offline:
  C:\Windows\system32\en-US\erofflps.txt
```
Additional detail: I compiled from source (de37ec19d6c4e490da55401158ed9c921c6ac547).

**Comment by td512** [comment]
Hey @darrik 

Are you able to run this in the debugger? It would tell me exactly what the issue is. I _think_ I know what's causing it, but if you wouldn't mind

**Comment by td512** [comment]
This has been fixed in [1.7.1](https://ltscdn.m6.nz/humble/trove-downloader.exe?version=1.7.1&utm_source=htd-github)
